codebase/config/sync/field.field.media.audio.field_media_of.yml

```diff
@@ -5,9 +5,16 @@ dependencies:
   config:
     - field.storage.media.field_media_of
     - media.type.audio
+    - node.type.islandora_object
```

Comment:
Hmmm...we might want this to include collections (collection_object) as well, so collections can have an image associated with them.

Reply:
(if that is what this is doing?)

Reply:
I just pushed a commit to add that